test/twitter/twits_test.exs

```diff
@@ -0,0 +1,99 @@
+defmodule Twitter.TwitsTest do
+  use TwitterWeb.ConnCase
+  # use Twitter.DataCase, async: true
+  # alias Twitter.Accounts.User
+  alias Twitter.Twits
+
+  describe "posts" do
+    alias Twitter.Twits.Post
+
+    @valid_attrs %{body: "some body"}
+    @update_attrs %{body: "some updated body"}
+    @invalid_attrs %{body: nil}
+
+    setup %{conn: conn} do
+      user = user_fixture()
+      conn = init_test_session(conn, current_user: user, user_id: user.id)
+      %{conn: conn, user: user}
+    end
+
+    test "list_posts/1 returns all posts of current user", %{user: user} do
+      post = post_fixture(user)
+      assert post = Twits.list_posts(user)
+    end
+
+    test "list_posts_by_author_id/1 returns all posts of a given author", %{user: user} do
+      # equivalent to list_posts when own id is passed
+      post = post_fixture(user)
+      assert post = Twits.list_posts_by_author_id(user.id)
+    end
+
+    test "get_post!/2 returns the post with given id", %{user: user} do
+      %Post{id: id} = post_fixture(user)
+      assert %Post{id: ^id} = Twits.get_post!(user, id)
+    end
+
+    test "create_post/2 with valid data creates a post", %{user: user} do
+      assert {:ok, %Post{} = post} = Twits.create_post(user, @valid_attrs)
+      assert post.body == "some body"
+    end
+
+    test "create_post/2 with invalid data returns error changeset", %{user: user} do
+      assert {:error, %Ecto.Changeset{}} = Twits.create_post(user, @invalid_attrs)
+    end
+
+    test "update_post/2 with valid data updates the post", %{user: user} do
+      post = post_fixture(user)
+      assert {:ok, %Post{} = post} = Twits.update_post(post, @update_attrs)
+      assert post.body == "some updated body"
```

Comment:
same as above

Reply:
I'm sorry I don't understand this.. Update post creates a changeset within and doesn't call the change_post function